views/js/checkout-component-renderer.js

```diff
@@ -67,351 +75,525 @@ jQuery(document).ready(function () {
 
-            componentBillingAddress = {
-                city: invoiceAddress.city,
-                country: invoiceAddress.country_iso,
-                houseNumberOrName: invoiceAddress.address2,
-                postalCode: invoiceAddress.postcode,
-                street: invoiceAddress.address1
+                componentDeliveryAddress = {
+                    city: deliveryAddress.city,
+                    country: deliveryAddress.country_iso,
+                    houseNumberOrName: deliveryAddress.address2,
+                    postalCode: deliveryAddress.postcode,
+                    street: deliveryAddress.address1,
+                };
             }
 
-            countryCode = invoiceAddress.country_iso;
-            phoneNumber = invoiceAddress.phone ? invoiceAddress.phone : invoiceAddress.phone_mobile;
+            componentPersonalDetails = {
+                firstName: prestashop.customer.firstname,
+                lastName: prestashop.customer.lastname,
+                shopperEmail: prestashop.customer.email,
+                telephoneNumber: phoneNumber,
+                gender: getAdyenGenderByPrestashopType(
+                    prestashop.customer.gender.type),
+                dateOfBirth: prestashop.customer.birthday,
+            };
         }
 
-        if (selectedDeliveryAddressId in prestashop.customer.addresses) {
-            var deliveryAddress = prestashop.customer.addresses[selectedDeliveryAddressId];
+        var configuration = Object.assign(
+            ADYEN_CHECKOUT_CONFIG,
+            {
+                hasHolderName: true,
+                holderNameRequired: false,
+                enableStoreDetails: !!isUserLoggedIn,
+                countryCode: countryCode,
+                data: {
+                    billingAddress: componentBillingAddress,
+                    deliveryAddress: componentDeliveryAddress,
+                    personalDetails: componentPersonalDetails,
+                },
+                onAdditionalDetails: handleOnAdditionalDetails,
+            },
+        );
 
-            componentDeliveryAddress = {
-                city: deliveryAddress.city,
-                country: deliveryAddress.country_iso,
-                houseNumberOrName: deliveryAddress.address2,
-                postalCode: deliveryAddress.postcode,
-                street: deliveryAddress.address1
-            }
-        }
+        window.adyenCheckout = new AdyenCheckout(configuration);
 
-        componentPersonalDetails = {
-            firstName: prestashop.customer.firstname,
-            lastName: prestashop.customer.lastname,
-            shopperEmail: prestashop.customer.email,
-            telephoneNumber: phoneNumber,
-            gender: getAdyenGenderByPrestashopType(prestashop.customer.gender.type),
-            dateOfBirth: prestashop.customer.birthday
-        }
-    }
+        // use this object to iterate through the stored payment methods
```

Comment:
Are these stored payment methods? Maybe it is a left over
```suggestion
        // use this object to iterate through the payment methods
```

Reply:
correct!